airbyte-api/src/main/openapi/config.yaml

```
@@ -1578,6 +1578,52 @@ paths:
                 $ref: "#/components/schemas/ImportRead"
         "404":
           $ref: "#/components/responses/NotFoundResponse"
+  /v1/source_oauths/set_instancewide_oauth_params:
```

Following naming conventions of other endpoints like `/v1/sources/create`, `/v1/destinations/create`, `/v1/connection/create` etc, should this be named as follow instead?

```suggestion
  /v1/source_oauths/oauth_params/create:
```

---

done